examples/react-instantsearch-hooks/tsconfig.node.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
is this used?

Reply:
Apparently not. It only served to reference vite.config.ts, which is automatically recognized by vite, so I removed the file, and it still works.